### PR TITLE
Close 2024 survey

### LINKED ISSUE #384: Close the 2024 user survey
Following #358 this issue group all action to stop advertising the survey
- [ ] Remove button on openrefine.org homepage
- [ ] Remove in-app notification
- [x] Unpin and close GitHub issue https://github.com/OpenRefine/OpenRefine/issues/6766
- [x] Remove the pinned [message from the forum](https://forum.openrefine.org/t/announcing-the-2024-openrefine-user-survey/1700/5)
- [ ] Close the all-our-idea platform for new vote
- [ ] Close the Google form for new entries

### PR DESCRIPTION
**Pull request by magdmartin**
Closes #384 

Remove the notification from the homepage and in-app notification

### CODE REVIEW

**Comment by magdmartin** [comment]
When updating line 160 of `index.js` to 
```
            $.getJSON("https://deploy-preview-385--openrefine-website.netlify.app/versions.json",
```
The notification to advertise the survey is no longer visible.

File: src/pages/usage.md [hunk 1]
```diff
@@ -52,27 +52,30 @@ xychart-beta
 In November 2022, we moved from email lists hosted by Google Groups to a Discourse forum. 
 
 As of March 20, 2024, over the last 12 months:
-* 244 new users signed up on our forum for a total of 470 users. 
-* 324 topics were created for a total of 1,700 messages.
+* 272 new users signed up[^1] on our forum for a total of 608 users[^2].
```
Comment (wetneb) [inline src/pages/usage.md:55]: Should the date above also be changed from March 20 to September 25?

Comment (magdmartin) [reply src/pages/usage.md:55]: this should have been fixed via #382

**Comment by magdmartin** [comment]
@wetneb are we good to merge this? I'd like to close the Google Form this week.

**Pull request merged**